### DISCUSSION TITLE
Customizable sensor for Daily Aux and add current,voltage and frequency values to AUX Load

### DISCUSSION THREAD

**Discussion by terzo33** [slipx06/sunsynk-power-flow-card#572] [Ideas]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Daily aux is calculated by card and i don't know why is always 0, but i prefer control daily load by my sensor like in grid option, then i can't see current,voltage and frequency values for aux load

### Expected behaviour

I need to calculate daily aux production by my accurate sensor daily. Then i need to see in my card "aux load" current,voltage and frequency values

### Possible Solutions

Add customizable daily aux production and in aux load add current,voltage and frequency values

![Screenshot_20241207_053141_Home Assistant](https://github.com/user-attachments/assets/37ef1725-8a64-4b2d-ae6f-bda154f1d314)

![Screenshot_20241207_054123_Home Assistant](https://github.com/user-attachments/assets/06300c5f-a2bc-4c63-b434-7e0ce94123c1)

### Mode

lite

### Context / Reason

Check accurate values

**Comment by slipx06** [comment]
Hi. Daily Aux is not calculated by the card. To display Daily Aux you need to provide a sensor i.e

`day_aux_energy: sensor.my_daily_aux`

**Comment by terzo33** [comment]
> Hi. Daily Aux is not calculated by the card. To display Daily Aux you need to provide a sensor i.e
> 
> `day_aux_energy: sensor.my_daily_aux`

do i have to manually create the sensor with that name or is there a section in the card where i can put my sensor?. And for current,voltage and frequency "AUX" values ?

Thanks

**Comment by slipx06** [comment]
Hi. You need to provide your own sensor for that entity. There is no option to display those other values at this stage. 

![image](https://github.com/user-attachments/assets/6cb39609-0089-43f5-9309-c9ac6f4f09b8)

**Comment by terzo33** [comment]
@slipx06 thanks a lot. Could you improve(current,voltage and frequency values for aux load) in next release?

**Comment by slipx06** [comment]
Not at the moment but moving this to an ideas discussion
